ui/libs/nh-launcher-applet/src/index.ts

```diff
@@ -17,31 +18,56 @@ export interface AppletBlock {
   render: Renderer;
 }
 
+export type ResourceView = (
+  rootElement: HTMLElement,
+  resourceHash: EntryHash,
```

Comment:
If the intention is for this to be an external interface that Applets have to implement, we will need to further genericise `resourceHash` to `string | number` and I would recommend renaming it something like `resourceIdentifier`, since different Resources will have different ways of being identified depending on how the Applet exposes them.

(eg. in hREA we expose `Unit.id` as a unique key scoped to the containing `DnaHash` and most other record IDs as `EntryHash:DnaHash`. Other hApps could have APIs which expose data by `AgentPubKey` or directly assigned strings- there are no standards for URI protocols for Holochain application data.)

Reply:
Thanks for pointing this out! I have some lingering notes about having a more generic approach to `resourceIdentifiers` (inspired by how I remember such implementations with hREA). I'm going to go ahead and make an issue for that soon.